src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/FileHistory.java

```diff
@@ -19,26 +20,35 @@
 @Setter
 @NoArgsConstructor
 public class FileHistory extends Auditable<String> {
-	
+
 	/**
 	 * Primary key
 	 */
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
 	@Id
 	@GeneratedValue
 	private Long fileHistoryId;
-		
-    /**
-     * The violation ticket number.
-     */
-    @Column(length = 50)
-    @Schema(nullable = false)
-    private String ticketNumber;
-    
-    /**
+
+	/**
+	 * The occam dispute id.
+	 */
+	@Column
+	@Schema(nullable = false)
+	@NotNull
+	private Long disputeId;
+
+	/**
+	 * File history entry type
+	 */
+	@Column(length = 4)
+	@Schema(nullable = false)
+	@NotNull
+	private AuditLogEntryType auditLogEntryType;
```

Comment:
By default, enums are serialized as the ordinal number (int). You may want to add `@Enumerated(EnumType.STRING)` to use the name of the enum as values in the database.

Reply:
This is true for storing the enum in H2 database as string but we don't store it in H2 anymore and only sending this to ORDS endpoint and get back from it. In this case, mapper takes care of converting it to the string value as auditLogEntryTypeCd. I haven't added the long descriptions for audit log entry enums for now. (some of them are really long) If we add that in the future, it can be useful to add toShortName() helper.

Reply:
Ah yes, right you are.